src/Avalonia.Controls/TextBlock.cs

```diff
@@ -348,14 +348,15 @@ public override void Render(DrawingContext context)
         /// Creates the <see cref="FormattedText"/> used to render the text.
         /// </summary>
         /// <param name="constraint">The constraint of the text.</param>
+        /// <param name="text">The text to generated the <see cref="FormattedText"/> for.</param>
```

Comment:
This comment doesn't make sense :) "The text to format." (or something similar) might be clearer